AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -3320,6 +3329,44 @@ private void displayRenderLoopDialog(Card mCurrentCard, RenderProcessGoneDetail
         }
     }
 
+    // show developer contact if js api used in card is deprecated
+    private void showDeveloperContact() {
+        View parentLayout = findViewById(android.R.id.content);
+        String snackbarMsg;
+        if (developerContact.equals("")) {
+            snackbarMsg = getString(R.string.api_version_no_developer_contact);
+        } else {
+            snackbarMsg = getString(R.string.api_version_developer_contact, developerContact);
+        }
+
+        Snackbar snackbar = Snackbar.make(parentLayout, snackbarMsg, 5000);
+        View snackbarView = snackbar.getView();
+        TextView snackTextView = (TextView) snackbarView.findViewById(com.google.android.material.R.id.snackbar_text);
```

Comment:
I believe there are auto-linting suggestions here that will let you remove `(TextView)`
```suggestion
        TextView snackTextView = (TextView) snackbarView.findViewById(com.google.android.material.R.id.snackbar_text);
```